src/v2/view-builder/shared/polling.js

```diff
@@ -0,0 +1,21 @@
+import { _ } from 'okta';
+
+export default {
+  addPolling () {
```

Comment:
nit: should this be called `startPolling` ? and similarly `stopPolling` instead of `removePolling`?